src/360-core/browzine-360-core-adapter.js

```diff
@@ -105,13 +107,27 @@ browzine.serSol360Core = (function() {
   function getCoverImageUrl(data) {
     var coverImageUrl = null;
 
-    if(data.coverImageUrl) {
-      coverImageUrl = data.coverImageUrl;
+    if(data) {
```

Comment:
Small nit, but could we do 
```
if (data && data.coverImageUrl) {
...
```
to save one level of nested `if`?

Reply:
Good one.